lib/serverengine/multi_spawn_server.rb

```diff
@@ -38,6 +38,14 @@ def initialize(worker_module, load_config_proc={}, &block)
       super(worker_module, load_config_proc, &block)
 
       @reload_signal = @config[:worker_reload_signal]
+      @pm.command_sender = @command_sender
+    end
+
+    def stop(stop_graceful)
+      if @command_sender == "pipe"
+        @pm.command_pipe.write stop_graceful ? "GRACEFUL_STOP\n" : "IMMEDIATE_STOP\n"
```

Comment:
I'm not sure about the order of priority of operators and method calls.
So it's better for me to use parenthesis.